docs/CHANGELOG.rst

```diff
@@ -23,6 +23,12 @@ Changed
 -------
 - **BACKWARD INCOMPATIBLE:** Remove comparison to reference in
   process ``mutations-table``
+- **BACKWARD INCOMPATIBLE:** Replace workflow
```

Comment:
- **BACKWARD INCOMPATIBLE:** Update workflow
   ``workflow-rnaseq-variantcalling``:

   - merge with functionality from deprecated workflow 
   ``workflow-rnaseq-variantcalling-beta``
   - add processes ``snpeff-single`` and ``mutations-table``
   - remove process ``gatk-select-variants-single``